rules/use-nunit-to-write-unit-tests/rule.md

```diff
@@ -0,0 +1,37 @@
+---
+type: rule
+title: Do you use NUnit to write Unit Tests?
+uri: use-nunit-to-write-unit-tests
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan/
+created: 2014-03-14T00:22:00.000Z
+guid: 63a5f8ba-42ad-4d32-aa65-7b9c1fdab799
+---
+When anyone sends you a bug that happen in their environment do a unit test. Just because the code runs on your machine it doesnt mean it will work on the users machine. E.g permissions issues - you are an admin while the user is only a simple user, registry & io reads might fail, NUnit will test for this and let you know.
+
+<!--endintro-->
+
+Unit testing is a valuable tool when maintaining code, particularly in a team environment where you may have to fix a bug in someone else's code. Unit Tests ensure that you do not introduce new bugs in someone else's code, or code that you have not looked at for a while. We like [NUnit](/ssw/Redirect/NUnit.htm) because it is free, we have found that it is easy for developers to learn and it integrates well with Visual Studio. Visual Studio .NET 2005 integrates Unit Testing with Visual Studio Team System. We will use this when Visual Studio 2005 is released.
```

Comment:
Hey @fenix2222 

- [ ] 1. Please update this NUnit link
- [ ] 2. I think we should remove the last sentence "We will use this when Visual Studio 2005 is released."